x-pack/plugins/alerting/server/usage/task.ts

```diff
@@ -111,10 +111,33 @@ export function telemetryTaskRunner(
               dailyExecutionTimeoutCounts,
               dailyFailedAndUnrecognizedTasks,
             ]) => {
+              const hasErrors =
+                totalCountAggregations.hasErrors &&
```

Comment:
this should be `||` instead of `&&` so hasErrors is true if any of these throw errors.